client/components/notice/test/index.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * External dependencies
+ */
+import { assert } from 'chai';
+import React from 'react';
+import { shallow } from 'enzyme';
+import { identity } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { Notice } from '../index';
+
+describe( 'index', function() {
```

Comment:
I'd either change this to something like `notice`, `Notice`, or `<Notice />`, else remove the block altogether. Dunno that `index` provides much benefit in the context of the test output.